content/deploy/cluster-sizing.md

```diff
@@ -1,29 +1,77 @@
 ---
-title: Cluster sizing
+title: Recommended pod sizing
 description: The recommended compute per pod
 ---
 
-The [Rhize Install]({{< relref "install" >}}) includes many services that run in their own pod.
-These services have different resource requirments.
+This document provides compute recommendations for the pods and services of your [Rhize Install]({{< relref "install" >}}).
+Some services also have recommended replication factors to increase reliability.
+
+## Node recommendations
+
+For your {{% param v %}} cluster, Rhize recommends a **minimum of three nodes** with the following specifications.
+
+### Rhize nodes
+
+
+| Property              | Value             |
+|-----------------------|-------------------|
+| Number of nodes       | 3                 |
+| CPU Speed (GHz)       | 3.3               |
+| vCPU per Node         | 16                |
+| Memory per node (GiB) | 32 (64 is better) |
+| Persisted volumes     | 12                |
+| PV IOPs               | 5000              |
+| PV Throughput (mbps)  | 500               |
```

Comment:
there is a subtle difference between MBps and mbps; mega bytes per second vs mega bits per second. In this instance we are refering to Mega Bytes per second. 

So please use MBps  or MB/s